README.md

```diff
@@ -1631,6 +1631,7 @@ _Related: [Web Servers](#web-servers)_
 - [imgproxy](https://imgproxy.net/) - Fast and secure standalone server for resizing and converting remote images. It works great when you need to resize multiple images on the fly without preparing a ton of cached resized images or re-doing it every time the design changes. ([Source Code](https://github.com/imgproxy/imgproxy)) `MIT` `Go/Docker`
 - [inlets](https://inlets.dev/) - Expose your local endpoints to the Internet - with a Kubernetes integration, Docker image and CLI available. `MIT` `Go/Docker`
 - [iodine](https://code.kryo.se/iodine/) - IPv4 over DNS tunnel solution, enabling you to start up a socks5 proxy listener. ([Source Code](https://github.com/yarrick/iodine)) `ISC` `C`
+- [Koblas](https://github.com/ynuwenhof/koblas) - Lightweight SOCKS5 proxy server. `MIT` `Rust`
```

Comment:
- Please change `Rust` to `Rust/Docker` 
- > Make sure the [Docker Engine](https://docs.docker.com/engine/install) is installed.

Reply:
Docker is no requirement since you can install and run the proxy server via cargo, nonetheless I am fine with adding Rust/Docker

Reply:
It may not be a requirement but its a way to install it, some people just look for docker installs and may miss this option if its not there. Thanks for adding it!